### PR TITLE
React2 week1 Assignment

### PR DESCRIPTION
**Pull request by patel-prakashkumar**
I have completed React2 Week1 Assignment with GitHub User Searcher and I haven't completed for Hyf Repos. I am working on Hyf user Repos context.

### CODE REVIEW

File: react/react2/week/github-user/src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Header from './Header';
+import Footer from './Footer';
+import GitUserContext from './GitUserContext';
```
Comment (annamourasobral) [inline react/react2/week/github-user/src/components/App.js]: Context isn't a component is just a way to pass the data through your components. 
```suggestion
import GitUserFetch from './GitUserFetch';
```

File: react/react2/week/github-user/src/components/GitUserContext.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { createContext } from "react";
+
+export const UserContext = createContext();
+
+export default function GitUserContext({ children }) {
+    const [user, setUser] = useState("");
```
Comment (annamourasobral) [inline react/react2/week/github-user/src/components/GitUserContext.jsx]: Only these states should be in the context, and the context should be in another file, Context.js, conventionally. Here you'll consume the context to fetch the data.

File: react/react2/week/github-user/src/components/GitUser.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+        <>
+        <div className="form-list">
+          <h2>GitHub Users List</h2>
+            {isLoading && <p>loading....</p>}
+            {error && <div><h2>Search is Failed : Error</h2><p>{error}</p></div>}
+            { usersResult.length === 0 && !error ? 
+                <h4>No results.</h4> : 
+                <ul>
+                    {usersResult.map((item => {
```
Comment (annamourasobral) [inline react/react2/week/github-user/src/components/GitUser.jsx]: Be specific 😉
```suggestion
                    {usersResult.map((user=> {
```

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hi, @patel-prakashkumar 
I can see your progress!
You met all the requirements, but your code needs retouching.
If you struggle with anything, just call me, okay?
Happy coding! 👩🏻‍💻

File: react/react2/week/github-user/src/components/GitUser.jsx (hunk 3 above)
Comment (patel-prakashkumar) [reply react/react2/week/github-user/src/components/GitUser.jsx]: Thank you for awesome feedback in assignment.

File: react/react2/week/github-user/src/components/App.js (hunk 1 above)
Comment (patel-prakashkumar) [reply react/react2/week/github-user/src/components/App.js]: I have updated according to feedback.

File: react/react2/week/github-user/src/components/GitUserContext.jsx (hunk 2 above)
Comment (patel-prakashkumar) [reply react/react2/week/github-user/src/components/GitUserContext.jsx]: I have created separate file for context and import to GitUserFetch file.

File: react/react2/week/github-user/src/components/GitUser.jsx (hunk 3 above)
Comment (patel-prakashkumar) [reply react/react2/week/github-user/src/components/GitUser.jsx]: I have change appropriate naming conversation.

**Review by annamourasobral — APPROVED** [review]

**Pull request merged**